### PR TITLE
Add Mentor Summit rules

### PR DESCRIPTION
**Pull request by rgaiacs**
I talked with @aterrel about this last December.

@aterrel Can you review the word of this document? And if it is OK, can you merge it?

@kain88-de Please let the mentors know this rule after @aterrel merge it.

### CODE REVIEW

File: organization/summit.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Mentor Summit
+
+Every year Google promote a Summit
+and each [**mentoring**] organization,
+**in this case only NumFOCUS**,
+may send two (2) members to the Mentor Summit.
+
+NumFOCUS will nominate as Summit ambassador
+the lead administrator of the current year
+**and** the lead administrator of following year.
```
Comment (kain88-de) [inline organization/summit.md]: I think it's a nice idea to allow the next year mentor to go as well. But I'm not sure how practical it is. For one only participants who sign up in the current year can come and Google wants people to sign up early in summer.  So there is half a year at least between a person agreeing to be the admin and actually having to admin. That's a lot of time for things to happen in life to not have time anymore.

Comment (rgaiacs) [inline organization/summit.md]: @kain88-de I agree with you. @aterrel and I hope that lead administrator stay for more than one edition so this rules only are used a few times. Reword suggestions are welcome.

Comment (kain88-de) [inline organization/summit.md]: I don't really know a better solution. I would give it a try and see if it works out. My idea to find a new admin would be to invite a member of the participating orgs if he wants to be the secondary admin and help me to look over the applications / student gradings. That way we have already experience with him and active project members should be likely to stay on GSoC for more then a year.

**Comment by kain88-de** [comment]
ping @aterrel

File: organization/summit.md (hunk 1 above)
Comment (aterrel) [inline organization/summit.md]: yes I would say that we have two admins now and if the two of them are interested in going they should have first pick.

**Comment by aterrel** [comment]
LGTM

**Pull request merged**